omnigibson/controllers/ik_controller.py

```diff
@@ -270,7 +270,10 @@ def _command_to_control(self, command, control_dict):
         target_joint_pos = self.solver.solve(
             target_pos=target_pos,
             target_quat=target_quat,
-            initial_joint_pos=current_joint_pos,
+            tolerance_pos=0.002,
+            weight_pos=20.0,
+            max_iterations=100,
+            # initial_joint_pos=current_joint_pos,
```

Comment:
why was this removed again?

Reply:
@mj-hwang what's happening here?

Reply:
This is due to some IK parameter tuning for skill. We can just revert it back to default.

Reply:
The commented line is there to ensure one-to-one matching during planning and execution time. Say the OMPL (skill) gives a set of waypoints. Even if we validated each planned state based on its previous state, we can’t guarantee all states’ validity during execution time since IK Controller does not exactly pass by all the planned states in a single step. When this undershoot/overshoot happens, the returned IK solutions change and we can't guarantee collision-freeness during the execution time.

Reply:
Not sure I fully understand Minjune's explanation. So are we reverting back? I don't think @cgokmen reverted back yet.